SPECS/ruby/ruby.spec

```diff
@@ -81,6 +81,9 @@ Version:        3.1.2
 Release:        3%{?dist}
 License:        (Ruby OR BSD) AND Public Domain AND MIT AND CC0 AND zlib AND UCD
 Vendor:         Microsoft Corporation
+# TODO: When changing ruby version, these gemified stdlib
+# provides should be versioned according to the ruby version.
+# More info: https://stdgems.org/
```

Comment:
Sorry for being picky, but could you place it right above the `Version` tag? It's more likely it's going to be noticed that way during updates.